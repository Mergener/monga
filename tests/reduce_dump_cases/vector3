type vec3 = {
	x : float;
	y : float;
	z : float;
};

type vec3array = [vec3];

function magnitude(v : vec3): float {
	return sqrt(v.x, v.y, v.z);
}