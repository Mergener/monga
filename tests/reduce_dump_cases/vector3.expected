type r1
def_field r1
field_defs r1
type r1
def_field r1
field_defs r2
type r1
def_field r1
field_defs r2
typedesc r3
def_type r1
definition r3
definitions r1
typedesc r1
typedesc r2
def_type r1
definition r3
definitions r2
type r1
parameter r1
parameters r1
opt_parameters r1
type r1
opt_ret_type r2
opt_variable_defs r1
var r1
exp_primary r3
var r2
exp_primary r3
exp_postfix r1
exp_unary r1
exp_multiplicative r1
exp_additive r1
exp_conditional r1
exp r1
exps r1
var r1
exp_primary r3
var r2
exp_primary r3
exp_postfix r1
exp_unary r1
exp_multiplicative r1
exp_additive r1
exp_conditional r1
exp r1
exps r2
var r1
exp_primary r3
var r2
exp_primary r3
exp_postfix r1
exp_unary r1
exp_multiplicative r1
exp_additive r1
exp_conditional r1
exp r1
exps r2
opt_exps r2
call r1
exp_primary r4
exp_postfix r1
exp_unary r1
exp_multiplicative r1
exp_additive r1
exp_conditional r1
exp r1
opt_exp r2
block r4
statements r1
opt_statements r2
block r1
def_function r1
definition r2
definitions r2
module r1
